packages/next/src/server/future/route-modules/app-page/vendored/ssr/entrypoints.ts

```diff
@@ -2,6 +2,8 @@ import * as React from 'react'
 import * as ReactDOM from 'react-dom'
 import * as ReactJsxDevRuntime from 'react/jsx-dev-runtime'
 import * as ReactJsxRuntime from 'react/jsx-runtime'
+//@ts-ignore TODO: current @types/react does not have exported types for this import
```

Comment:
nit: we're trying to move away from `@ts-ignore`, because it doesn't yell at you when it's no longer necessary
```suggestion
// @ts-expect-error TODO: current @types/react does not have exported types for this import
```